BaseTools/Plugin/WindowsVsToolChain/WindowsVsToolChain.py

```diff
@@ -204,12 +204,19 @@ def do_pre_build(self, thebuilder):
                         HostType = "x86"
                     elif HostInfo.bit == "64":
                         HostType = "x64"
+                # MU_CHANGE: Support Windows-ARM
+                elif HostInfo.arch == "ARM":
+                    if HostInfo.bit == "32":
+                        HostType = "arm"
+                    elif HostInfo.bit == "64":
+                        HostType = "arm64"
+                # MU_CHANGE
                 else:
                     raise NotImplementedError()
 
             # VS2022_HOST options are not exactly the same as QueryVcVariables. This translates.
             VC_HOST_ARCH_TRANSLATOR = {
-                "x86": "x86", "x64": "AMD64", "arm": "not supported", "arm64": "not supported"}
+                "x86": "x86", "x64": "AMD64", "arm": "x86_arm", "arm64": "amd64_arm64"} # MU_CHANGE: Support Windows-ARM

```

Comment:
do we care about `x86_arm64` or `amd64_arm`?

Reply:
Given how we populated the 2 fields for x86 and x64, i don't think we do. But I will leave this comment open for others to chime in.